Maquetacion/index.html

```diff
@@ -91,5 +91,10 @@ <h2 class="text title-card">Django</h2>
         </div>
       </div>
     </div>
+    <footer class="footer-bar">
+      <p class="text-f">we are game changers</p>
```

Comment:
I would recommend using a more descriptive name on the classes so it can be easily understood just by looking at the CSS file. :)

Reply:
class is changed to footer-text